actions/mongodb_explain_action_test.go

```diff
@@ -0,0 +1,76 @@
+package actions
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/percona/percona-toolkit/src/go/mongolib/proto"
+	"github.com/percona/pmm-agent/utils/tests"
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func TestMongoDBExplain(t *testing.T) {
+	database := "test"
+	collection := "test_col"
+	id := "abcd1234"
+	ctx := context.TODO()
+
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI(tests.MongoDBDSN()))
+	if err != nil {
+		t.Fatalf("Cannot connect to MongoDB: %s", err)
+	}
```

Comment:
In general, we use `require.NoError(t, err)` in such cases.

Reply:
It's not addressed yet